Makefile.PL

```diff
@@ -12,7 +12,6 @@ all_from 'lib/Zonemaster/CLI.pm';
 tests_recursive( 't' );
 
 resources(
-    license    => 'http://www.opensource.org/licenses/BSD-3-Clause',
```

Comment:
Shouldn't it rather be

license => 'https://opensource.org/licenses/BSD-2-Clause'

Reply:
No, the value of the resources/license property is overridden by a value derived from the license property when META.yml is generated. When license is set to "bsd" the URL that ends up in META.yml becomes "http://opensource.org/licenses/bsd-license.php". That URL points to a BSD-2 template.